invenio_cli/cli/cli.py

```diff
@@ -141,4 +141,15 @@ def destroy(cli_config):
 @invenio_cli.command()
 def upgrade():
     """Upgrades the current application to the specified newer version."""
-    raise click.UsageError("Not supported yet.")
+    os.popen('pipenv run invenio alembic upgrade')
```

Comment:
major: I am not sure I understand this...
- There are two functions with the same code inside, what's the reason behind it?
- The `upgrade` function is in the correct place... However, it should be done in the same fashion than the packages cli you implemented (e.g. `outdated`). This means, create a function at `command` level that return `CommandStep`s and son on... It should not invoke any `os` function at this level.

Reply:
I forgot to rollback this, but the upgrade wasn't implemented in this draft. It wast just old code.